examples/llava/clip.cpp

```diff
@@ -13,6 +13,10 @@
 #define STB_IMAGE_IMPLEMENTATION
 #include "stb_image.h"
 
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
+#endif
```

Comment:
@ggerganov just want to double check with you, do you think it's ok to copy this def to all examples?

Reply:
We could move them to some appropriate local header, of course, if that is preferred, provided that header will be included wherever needed.

Reply:
I'm not familiar with `__STDC_FORMAT_MACROS` - what is it used for? I don't think we should add it if it's only purpose is to make some ancient compilers work.

Reply:
@ggerganov Revert issue on AIX, for example: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=97044

Reply:
Based on the mentioned thread, I think it's already fixed at some point, right?

IMO it's not intuitive to include this def everywhere in the code base, especially when most developers can't even find a good source of truth for what it does.

Also, do this fix actually fix an issue that you're having, or just something you just know about and fix (like nits fixing?)

Reply:
For me personally using `cinttypes` is sufficient, but `inttypes.h` requires `__STDC_FORMAT_MACROS` (which is why I got the error with the build).
If using `__STDC_FORMAT_MACROS` is controversial, just switching to C++ header is good enough.